Holovibes/includes/core/icompute.hh

```diff
@@ -291,47 +138,13 @@ class ICompute
     {
         spdlog::trace("[ICompute] [update_setting] {}", typeid(T).name());
 
-        if constexpr (has_setting<T, decltype(realtime_settings_)>::value)
-        {
+        if constexpr (has_setting_v<T, decltype(realtime_settings_)>)
```

Comment:
Here you are updating by removing 'if constexpr (has_setting<T, decltype(realtime_settings_)>::value)'. However this line can still be found in converts.hh, fourier_transform.hh, image_accumulation.hh, etc....

It is the same for multiple of the changes you made by removing has_setting lines by has_setting_v. Check for all the occurency in the project.